atom/browser/ui/cocoa/atom_touch_bar.mm

```diff
@@ -145,7 +136,30 @@ - (void)refreshTouchBarItem:(NSTouchBar*)touchBar
   } else if (item_type == "scrubber") {
     [self updateScrubber:(NSCustomTouchBarItem*)item withSettings:settings];
   }
+}
 
+- (void)refreshTouchBarItem:(NSTouchBar*)touchBar
+                         id:(const std::string&)item_id {
+  if (![self hasItemWithID:item_id]) return;
+
+  mate::PersistentDictionary settings = settings_[item_id];
+  std::string item_type;
+  settings.Get("type", &item_type);
+
+  NSTouchBarItemIdentifier identifier = [self identifierFromID:item_id
+                                                          type:item_type];
+  if (!identifier) return;
+
+  std::vector<std::string> popover_ids;
+  if (settings.Get("_popover", &popover_ids)) {
+    for (size_t i = 0; i < popover_ids.size(); ++i) {
```

Comment:
I think you could use `for (auto& popover_id : popover_ids)` here since it looks like `i` isn't needed.